### PR TITLE
Two new inscription editions and an attempt at translating them

### PR DESCRIPTION
**Pull request by noah-cashian**
Might have messed up the pull request again - another abortive test?

### CODE REVIEW

File: inscriptions/ISic000543.xml [hunk 1]
```diff
@@ -187,13 +192,26 @@
...
+                    <lb n="2"/> 
+                        conivgi
+                    <lb n="3"/> 
+                        sancitissimae
+                    <lb n="4"/> 
+                        <expan><abbr>P</abbr><ex>vblivs</ex></expan> 
+                            <g ref='#interpunt'/> 
+                                Aelivs 
+                                    <g ref='#interpunt'/> 
+                                    <expan><abbr>A</abbr><ex>vgvsti</ex></expan>
```
Comment (JonPrag) [inline inscriptions/ISic000543.xml:210]: I think it's Aug(usti)?

File: inscriptions/ISic000543.xml [hunk 2]
```diff
@@ -202,8 +220,10 @@
                 <app loc="line 5"><note>Auria 1697: [..]IGR[..]NVS[..]; Gualtherus 1624: ΛIGRANIVS</note></app>
                 </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div xml:lang='en' type="translation" resp='#NCash'>
+                <p>Publius Aelius Nigrianus, freedman of the emperor, you enlarge (this) for his most sacred wife Aelia</p>
+                <!--Latin Q - 'auge' imperative seems awkward here, or am I just unused to epigraphic Latin?-->
```
Comment (JonPrag) [inline inscriptions/ISic000543.xml:225]: surely a second name 'Auge' (Greek name): [http://clas-lgpn2.classics.ox.ac.uk/cgi-bin/lgpn_search.cgi?namenoaccents=%CE%91%CE%A5%CE%93%CE%97](http://clas-lgpn2.classics.ox.ac.uk/cgi-bin/lgpn_search.cgi?namenoaccents=%CE%91%CE%A5%CE%93%CE%97)

File: inscriptions/ISic000543.xml [hunk 3]
```diff
@@ -202,8 +220,10 @@
...
+                <!--If Aug(usta) (libert), then: 'Publius Aelius Nigrianus, freedman of the emperor, (made) (this) for his most sacred wife Aelia, freedwoman of the emperor'-->
```
Comment (JonPrag) [inline inscriptions/ISic000543.xml:226]: chaste or virtuous (or pure), rather than 'sacred'.

File: inscriptions/ISic000543.xml [hunk 4]
```diff
@@ -202,8 +220,10 @@
                 <app loc="line 5"><note>Auria 1697: [..]IGR[..]NVS[..]; Gualtherus 1624: ΛIGRANIVS</note></app>
```
Comment (JonPrag) [inline inscriptions/ISic000543.xml:220]: which leaves us with an interesting question, as to whether we think it's Nigranius or Nigrianus; the latter is however better attested from a quick look in Clauss Slaby.

**Review by JonPrag — CHANGES_REQUESTED** [review]
Dear Noah

Absolutely the right idea and broadly correct - well done. The issues here are almost all tiny peccadilloes (and all are noted in comments individually). In writing code the precision is however important: from a purely syntactical perspective in XML, the spaces must only be in the right places inside the tags; values on attributes must be in "" not in ' '; and a value on a ref needs the # on it. In the text, could you use u and v, rathe rthan just v; and could you aim to close up all space, except between individual words?
As I say, all very small things. Once the corrections are made commit them and repeat the pull request process, which will simply add them tot he existing pull request, and then I will approve and incorporate. 
many thanks!
Jonathan

**Comment by noah-cashian** [comment]
Dear Jonathan,

Sorry for those silly mistakes, and thank you for pointing out where they
were. I've just made a new commit for both files and have pushed it to the
origin from my desktop GitHub app, although I'm not sure if this has been
automatically merged with the original pull request. Just in case, I
thought I'd email you to let you know.

All best,
Noah

**Review by JonPrag — APPROVED** [review]
Excellent, many thanks - use of spaces all looks good now

**Pull request merged**